src/components/donationCard/DonationCard.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import ReactDom from "react-dom";
+import "../donationCard/DonationCard.scss";
+import {
+  FaHandHoldingHeart,
+  FaIdeal,
+  FaRegCalendarCheck,
+} from "react-icons/fa";
+import { GiPiggyBank } from "react-icons/gi";
+import Button from "../BtnCtA";
+import axios from "axios";
+//import { formatDistance } from "date-fns";
```

Comment:
We can get rid of this commented out code (and other commented out code in this file)